src/Engine/GameInstance.java

```diff
@@ -218,20 +222,43 @@ private String getSaveName()
     return sb.toString();
   }
   
+  public static boolean isSaveCompatible(String filename)
+  {
+    System.out.println(String.format("Checking compatibility of save %s", filename));
```

Comment:
If it fails as incompatible, it would be good to print out the version of the save so we can go find the right DefendPeace version if we want.

Reply:
We do, when we catch the exception.

Reply:
...on the other method. Fair point.